plugins/feedback/src/components/FeedbackDetailsModal/FeedbackDetailsModal.tsx

```diff
@@ -0,0 +1,356 @@
+              justifyContent="space-between"
+              alignItems="center"
+              spacing={2}
+            >
+              <Grid item xs={12}>
+                <Typography className={classes.submittedBy} variant="body2">
+                  Submitted by&nbsp;
+                  <EntityRefLink entityRef={modalData.createdBy}>
+                    {parseEntityRef(modalData.createdBy).name}
+                  </EntityRefLink>{' '}
+                  on {new Date(modalData.createdAt).toLocaleString()}
+                </Typography>
+              </Grid>
+              <Grid item xs={12}>
+                <Typography variant="body1">
+                  {modalData.description
+                    ? getDescription(modalData.description)
+                    : 'No description provided'}
+                </Typography>
+                {modalData.description.length > 400 ? (
+                  <Link
+                    className={classes.readMoreLink}
+                    onClick={() => setExpandDescription(!expandDescription)}
+                  >
+                    {!expandDescription ? (
+                      <ExpandMoreRounded />
+                    ) : (
+                      <ExpandLessRounded />
+                    )}
+                    {!expandDescription ? 'Read More' : 'Read Less'}
+                  </Link>
+                ) : null}
+              </Grid>
+              <Grid item xs={12}>
+                <List title="feedback-details-list" disablePadding>
+                  <ListItem>
+                    <ListItemText
+                      primary={
+                        modalData.feedbackType === 'FEEDBACK'
+                          ? 'Feedback submitted for'
+                          : 'Issue raised for'
+                      }
+                    />
+                    <ListItemSecondaryAction>
+                      <ListItemText
+                        primary={
+                          <EntityRefLink entityRef={modalData.projectId}>
+                            <Chip
+                              clickable
+                              variant="outlined"
+                              color="primary"
+                              label={
+                                modalData.projectId.split('/').slice(-1)[0]
+                              }
+                            />
+                          </EntityRefLink>
+                        }
+                      />
+                    </ListItemSecondaryAction>
+                  </ListItem>
+                  <ListItem>
+                    <ListItemText primary="Tag" />
+                    <ListItemSecondaryAction>
+                      <ListItemText
+                        primary={
+                          <Chip variant="outlined" label={modalData.tag} />
+                        }
+                      />
+                    </ListItemSecondaryAction>
+                  </ListItem>
+                  {modalData.ticketUrl && (
```

Comment:
slight nit but it's safer to use a ternery operator for conditional rendering, in some cases this style can [render unexpected results](https://dev.to/maafaishal/avoid-operator-for-conditional-rendering-in-react-2de).